packages/Molecules/Card/CardCarousel.vue

```diff
@@ -0,0 +1,210 @@
+<template>
+  <div class="rpl-carousel">
```

Comment:
Could you please correct the class name to match the component name? It should be `rpl-card-carousel`.

Reply:
Fixed.